std/file.d

```diff
@@ -3831,9 +3860,15 @@ unittest
 
 unittest
 {
-    import std.algorithm;
-    import std.range;
-    import std.process;
+    import std.algorithm.searching : startsWith;
+    import std.algorithm.comparison : equal;
+    import std.algorithm.iteration : map;
+    import std.range.primitives : walkLength;
+    import std.process : thisProcessID;
+    import std.path : dirEntries, buildPath, absolutePath;
+    import std.conv : to;
+    import std.array : array;
```

Comment:
As long as you're greatly expanding this list of imports, please alphabetically order them.